mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -3444,6 +3444,71 @@ def NVVM_Tcgen05StOp : NVVM_Op<"tcgen05.st"> {
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// NVVM dp4a Op
+//===----------------------------------------------------------------------===//
+
+def DP4aS8 : I32EnumAttrCase<"S8", 1, "s8">;
+def DP4aU8 : I32EnumAttrCase<"U8", 0, "u8">;
```

Comment:
We've types in nvvm dialects for wgmma, and maybe some other ops. We should unify them. it's okay for this PR, but it's better if we can do it sooner